seismicpro/survey/utils.py

```diff
@@ -39,3 +41,29 @@ def ibm_to_ieee(hh, hl, lh, ll):
             exp16 = (np.int8(hh[i, j]) & np.int8(0x7f)) - 70
             res[i, j] = mant * 16.0**exp16
     return res
+
+
+def mute_and_norm(gather, muter_col=HDR_FIRST_BREAK, rv_params=None):
+    """Mute direct wave using `first_breaks_col` and normalise"""
+    if muter_col not in gather.headers:
+        raise RuntimeError(f"{muter_col} not in headers")
+
+    if rv_params is None:
+        rv_params = dict(n_refractors=1)
+
+    muter = gather.calculate_refractor_velocity(first_breaks_col=muter_col, **rv_params).create_muter()
+    return gather.copy().mute(muter=muter, fill_value=np.nan).scale_standard()
+
+
+@njit
+def rms_2_windows_ratio(data, n_begs, s_begs, win_size):
+    """Compute RMS ratio for 2 windows defined by their starting samples and window size."""
+    res = np.full(data.shape[0], fill_value=np.nan)
+
+    for i, (trace, n_beg, s_beg) in enumerate(zip(data, n_begs, s_begs)):
```

Comment:
Consider to add parallelization
```suggestion
    for i in prange(len(res)):
        trace, n_beg, s_beg = data[i], n_begs[i], s_begs[i]
```